test.py

```diff
@@ -12,6 +12,8 @@
 import argparse
 import asyncio
 import collections
+from random import randint
```

Comment:
> Fix LDAP tests flakiness due not possible to connect to LDAP server
> Add catching stdout and stderr of toxiproxy-cli in case of errors

in future, would be better to use dedicated commit for "moving" code around, and use separate commit to implement functional changes.

Reply:
Because it's removing starting the proxy for each test. Now, tests using the same proxy. Additionally, toxiproxy server starts on a separate host, so no occupied ports will be there.